config/webpack/environment.js

```diff
@@ -1,4 +1,18 @@
 const { environment } = require("@rails/webpacker")
+const dotenv = require("dotenv")
+const webpack = require("webpack")
+
+const dotenvFiles = [".env"]
+dotenvFiles.forEach((dotenvFile) => {
```

Comment:
What do these changes do?

Reply:
They are loading env variables. It seems like they are not loaded automatically.